Languages/Java/answers.json

```diff
@@ -15,6 +15,18 @@
     {
       "answer": "It is the polymorphism technique which allows us to create multiple methods with the same name but different signature.",
       "details": "Method Overloading increases the readability of the program. Method overloading is performed to figure out the program quickly."
+    },
```

Comment:
Omega Good Job!

@prasad-more-12  what other languages do you know?

Reply:
I'd prefer if you actually added to the code :D It is a hackathon afterall. Feel free to either pick up a ticket  or open the project and look for changes yourself :D 

To build the project, open the Interview_Prep_App folder with your IDE.